doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -116,17 +119,30 @@ of untrusted certificates associated with I<ctx> to I<sk>.
 X509_STORE_CTX_verify() will take the first element, if any,
 as its default target if the target certificate is not set explicitly.
 
-X509_STORE_CTX_set0_param() sets the internal verification parameter pointer
-to I<param>. After this call B<param> should not be used.
+X509_STORE_CTX_get_num_untrusted() returns the number of untrusted certificates
+that were used in building the chain following a call to L<X509_verify_cert(3)>.
+
+X509_STORE_CTX_get_num_untrusted() returns the number of untrusted certificates
```

Comment:
These lines are repeated of the above.  Remove the above?

Reply:
Oops, yes.